frank/utilities.py

```diff
@@ -259,6 +267,8 @@ def apply_correction_to_weights(u, v, ReV, weights, nbins=300): # TODO: should c
         :math:`1 / \sigma^2`
     """
 
+    logging.info('  Estimating, applying correction factor to visibility weights') # TODO: should go in utilities.py
+
```

Comment:
This should be left here, or if moved to utlilities.py, make it debug level

Reply:
I don't see this as unnecessary but as a useful status report to the user. I'd prefer to leave at the info level.

Reply:
I feel the same about this one as load_uvtable

Reply:
Moved to fit.py